main.pm

```diff
@@ -385,6 +389,9 @@ sub load_inst_tests() {
         else {
             loadtest "installation/skip_registration.pm";
         }
+        if (is_sles4sap) {
```

Comment:
oh, we're in the link exchange game - http://www.perlmonks.org/?node_id=989115#11